### ISSUE TITLE
Exception in current Idea 

### ISSUE THREAD

**Issue by wirepatch** [dmarks2/maven-archetypes-catalog-plugin#13]
Environment: IntelliJ 2020.1.1 April 30 on Ubuntu 18.04 LTS 

Reproducibly observing:
```
Should be called at least in the state COMPONENTS_REGISTERED, the current state is: LAF_INITIALIZED
Current violators count: 1



java.lang.Throwable
	at com.intellij.diagnostic.LoadingState.logStateError(LoadingState.java:56)
	at com.intellij.diagnostic.LoadingState.checkOccurred(LoadingState.java:52)
	at com.intellij.serviceContainer.BaseComponentAdapter.getInstanceUncached(BaseComponentAdapter.kt:73)
	at com.intellij.serviceContainer.BaseComponentAdapter.getInstance(BaseComponentAdapter.kt:69)
	at com.intellij.serviceContainer.ComponentManagerImpl.doGetService(ComponentManagerImpl.kt:411)
	at com.intellij.serviceContainer.ComponentManagerImpl.getService(ComponentManagerImpl.kt:394)
	at com.intellij.openapi.components.ServiceManager.getService(ServiceManager.java:20)
	at com.intellij.ide.plugins.PluginUtil.getInstance(PluginUtil.java:11)
	at com.intellij.notification.NotificationGroup.<init>(NotificationGroup.kt:43)
	at com.intellij.notification.NotificationGroup.<init>(NotificationGroup.kt:28)
	at com.intellij.notification.NotificationGroup.<init>(NotificationGroup.kt:36)
	at com.intellij.notification.NotificationGroup.<init>(NotificationGroup.kt:36)
	at com.intellij.notification.NotificationGroup.<init>(NotificationGroup.kt)
	at de.dm.intellij.maven.archetypes.plugin.ArchetypeCatalogApplicationComponent.<clinit>(ArchetypeCatalogApplicationComponent.java:17)
	at java.base/java.lang.Class.forName0(Native Method)
	at java.base/java.lang.Class.forName(Class.java:398)
	at com.intellij.serviceContainer.ComponentManagerImpl.registerComponent(ComponentManagerImpl.kt:288)
	at com.intellij.serviceContainer.ComponentManagerImpl.registerComponents(ComponentManagerImpl.kt:194)
	at com.intellij.serviceContainer.ComponentManagerImpl.registerComponents(ComponentManagerImpl.kt:157)
	at com.intellij.idea.ApplicationLoader$registerAppComponents$1.apply(ApplicationLoader.kt:93)
	at com.intellij.idea.ApplicationLoader$registerAppComponents$1.apply(ApplicationLoader.kt)
	at java.base/java.util.concurrent.CompletableFuture.uniApplyNow(CompletableFuture.java:680)
	at java.base/java.util.concurrent.CompletableFuture.uniApplyStage(CompletableFuture.java:658)
	at java.base/java.util.concurrent.CompletableFuture.thenApply(CompletableFuture.java:2094)
	at com.intellij.idea.ApplicationLoader.registerAppComponents(ApplicationLoader.kt:91)
	at com.intellij.idea.ApplicationLoader.executeInitAppInEdt(ApplicationLoader.kt:63)
	at com.intellij.idea.ApplicationLoader.access$executeInitAppInEdt(ApplicationLoader.kt:1)
	at com.intellij.idea.ApplicationLoader$initApplication$1$1.run(ApplicationLoader.kt:355)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:313)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:776)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:727)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:746)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
```

**Comment by midorlo** [comment]
Also traced from StartupUtil

```
java.lang.Throwable
	at com.intellij.diagnostic.LoadingState.logStateError(LoadingState.java:56)
	at com.intellij.diagnostic.LoadingState.checkOccurred(LoadingState.java:52)
	at com.intellij.serviceContainer.BaseComponentAdapter.getInstanceUncached(BaseComponentAdapter.kt:73)
	at com.intellij.serviceContainer.BaseComponentAdapter.getInstance(BaseComponentAdapter.kt:69)
	at com.intellij.serviceContainer.ComponentManagerImpl.doGetService(ComponentManagerImpl.kt:411)
	at com.intellij.serviceContainer.ComponentManagerImpl.getService(ComponentManagerImpl.kt:394)
	at com.intellij.openapi.components.ServiceManager.getService(ServiceManager.java:20)
	at com.intellij.ide.plugins.PluginUtil.getInstance(PluginUtil.java:11)
	at com.intellij.notification.NotificationGroup.<init>(NotificationGroup.kt:43)
	at com.intellij.notification.NotificationGroup.<init>(NotificationGroup.kt:28)
	at com.intellij.notification.NotificationGroup.<init>(NotificationGroup.kt:36)
	at com.intellij.notification.NotificationGroup.<init>(NotificationGroup.kt:36)
	at com.intellij.notification.NotificationGroup.<init>(NotificationGroup.kt)
	at de.dm.intellij.maven.archetypes.plugin.ArchetypeCatalogApplicationComponent.<clinit>(ArchetypeCatalogApplicationComponent.java:17)
	at java.base/java.lang.Class.forName0(Native Method)
	at java.base/java.lang.Class.forName(Class.java:398)
	at com.intellij.serviceContainer.ComponentManagerImpl.registerComponent(ComponentManagerImpl.kt:288)
	at com.intellij.serviceContainer.ComponentManagerImpl.registerComponents(ComponentManagerImpl.kt:194)
	at com.intellij.serviceContainer.ComponentManagerImpl.registerComponents(ComponentManagerImpl.kt:157)
	at com.intellij.idea.ApplicationLoader$registerAppComponents$1.apply(ApplicationLoader.kt:93)
	at com.intellij.idea.ApplicationLoader$registerAppComponents$1.apply(ApplicationLoader.kt)
	at java.base/java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:642)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506)
```
```
	at java.base/java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2073)
	at com.intellij.idea.ApplicationLoader$initApplication$2.accept(ApplicationLoader.kt:379)
	at com.intellij.idea.ApplicationLoader$initApplication$2.accept(ApplicationLoader.kt)
	at java.base/java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:859)
	at java.base/java.util.concurrent.CompletableFuture.uniWhenCompleteStage(CompletableFuture.java:883)
	at java.base/java.util.concurrent.CompletableFuture.whenComplete(CompletableFuture.java:2251)
	at java.base/java.util.concurrent.CompletableFuture.whenComplete(CompletableFuture.java:143)
	at com.intellij.idea.ApplicationLoader.initApplication(ApplicationLoader.kt:376)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at com.intellij.idea.MainImpl$1.proceed(MainImpl.java:104)
	at com.intellij.ide.b.b.v.proceed(v.java:90)
	at com.intellij.ide.b.b.aQ.b(aQ.java:260)
	at com.intellij.ide.b.b.a7.b(a7.java:276)
	at com.intellij.ide.b.b.a7.c(a7.java:169)
	at com.intellij.ide.b.b.a7.a(a7.java:78)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at com.intellij.idea.MainImpl.start(MainImpl.java:96)
	at com.intellij.idea.StartupUtil.startApp(StartupUtil.java:289)
	at com.intellij.idea.StartupUtil.prepareApp(StartupUtil.java:233)
	at com.intellij.ide.plugins.MainRunner.lambda$start$0(MainRunner.java:45)
	at java.base/java.lang.Thread.run(Thread.java:834)
```

**Comment by dmarks2** [comment]
Bug is fixed in version 1.4.0.
